libs/filabridge/include/private/filament/UibStructs.h

```diff
@@ -65,7 +65,7 @@ struct PerViewUib { // NOLINT(cppcoreguidelines-pro-type-member-init)
     math::float4 sun; // cos(sunAngle), sin(sunAngle), 1/(sunAngle*HALO_SIZE-sunAngle), HALO_EXP
 
     math::float2 lightFarAttenuationParams;     // a, a/far (a=1/pct-of-far)
-    float padding0;
+    float needsAlphaChannel;
```

Comment:
Does this need to be a float? Could we use a single bit instead?

Reply:
It's just that ubos can't have bitfields it bools